components/resources/wallet_strings.grdp

```diff
@@ -646,6 +646,9 @@
   <message name="IDS_BRAVE_WALLET_TRY_SEARCHING_FOR_DIFFERENT_ASSET" desc="Hint for no provider found for buy asset">Try searching for a different asset.</message>
   <message name="IDS_BRAVE_WALLET_NO_RESULTS_FOUND" desc="No search results for provider title">No results found for <ph name="PROVIDER">$1<ex>Ramp</ex></ph></message>
   <message name="IDS_BRAVE_WALLET_TRY_DIFFERENT_KEYWORDS" desc="Hint for no search result for provider">Try using a different keyword or check your spelling.</message>
+  <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_TO_BUY_TITLE" desc="Create account modal title">Do you want to create a <ph name="ACCOUNT_TYPE">$1<ex>Zcash</ex></ph> Account?</message>
```

Comment:
Should the A in account be uppercased?

Reply:
```suggestion
  <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_TO_BUY_TITLE" desc="Create account modal title">Do you want to create a <ph name="ACCOUNT_TYPE">$1<ex>Zcash</ex></ph> account?</message>
```

Reply:
I think so since it is a `proper noun` in the sentence.

![Screenshot 18](https://github.com/user-attachments/assets/b2d844a7-b9c2-4792-b2bd-73d3e3e63b70)